tests/integration_tests/weekly_rand_multi_failover/run.sh

```
@@ -0,0 +1,196 @@
+function cleanup() {
+	set +e
+	cleanup_process $CDC_BINARY >/dev/null 2>&1 || true
+	cleanup_process cdc_kafka_consumer >/dev/null 2>&1 || true
+	cleanup_process cdc_storage_consumer >/dev/null 2>&1 || true
+	cleanup_process cdc_pulsar_consumer >/dev/null 2>&1 || true
+	stop_test $WORK_DIR
+}
+
+trap 'cleanup' EXIT
+
+rm -rf $WORK_DIR && mkdir -p $WORK_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)